.github/environments/leafcloud-ha/inventory/group_vars/all/variables.yml

```
@@ -35,13 +35,12 @@ capi_cluster_addons_csi_cinder_volume_type: unencrypted
 # The risk of failed upgrades is too great, and it is going away soon
 consul_server_replicas: 1
 
-# Enable Velero just to check that installation works
-velero_enabled: true
-velero_s3_url: https://required-but-not-used.com
-velero_bucket_name: not-used
-velero_backup_schedule_enabled: true
-velero_backup_schedule_name: default
-velero_backup_schedule_timings: "0 0 * * *"
-velero_backup_schedule_ttl: "168h"
-velero_aws_access_key_id: required-but-not-used
-velero_aws_secret_access_key : required-but-not-used
+# Pick up the reserved IP for the Zenith SSHD LB
+zenith_sshd_service_load_balancer_ip: "{{ lookup('env', 'ZENITH_SSHD_IP') }}"
+
+# Configure Velero backups
+velero_enabled: "{{ not (not velero_aws_access_key_id) }}"
```

Nit: I think this is would be cleaner
```suggestion
velero_enabled: "{{ velero_aws_access_key_id | bool }}"
```